tests/testCoadd.py

```diff
@@ -43,21 +42,18 @@
     display
 except NameError:
     display = False
-    Verbosity = 0 # increase to see trace
+    Verbosity = 0  # increase to see trace
 
 pexLog.Trace_setVerbosity("lsst.coadd.utils", Verbosity)
 
 try:
     AfwDataDir = lsst.utils.getPackageDir('afwdata')
+    ImSimFile = os.path.join(AfwDataDir, "ImSim/calexp/v85408556-fr/R23/S11.fits")
```

Comment:
Same comment here as previously. Also, `AfwDataDir` is inconsistent with other test calling it `dataDir`. I think we could just use `dataDir` as the variable everywhere.